src/airflow_declarative/trafaret.py

```diff
@@ -188,6 +189,23 @@ def cast_interval(value):
     return value
 
 
+def cast_crontab_or_interval(value):
+    try:
+        return cast_interval(value)
+    except ValueError:
```

Comment:
How `ValueError` happens there? I see only DataError raised in those function. Let's catch it since we don't know if this exception will remain subclass of ValueError. I would like to separate excepted exception from bug here.

Reply:
IIRC I just chose the base class of DataError. Personally I don't see the difference, so I've just changed it to DataError.